.eleventy.js

```diff
@@ -2,41 +2,49 @@ const fetch = require("node-fetch");
 const parseGitUrl = require("git-url-parse");
 
 module.exports = function (config) {
-
   // Add filters to Nunjucks
-  config.addFilter("squash", require("./filters/squash.js") );
-  config.addFilter("appendToName", require("./filters/fileNameAppend.js") );
-  config.addFilter("stringify", require("./filters/stringify.js") );
-  config.addFilter("capitalize", (text) => text.replace(/\b\w/g, (m) => m.toUpperCase()));
+  config.addFilter("squash", require("./filters/squash.js"));
+  config.addFilter("appendToName", require("./filters/fileNameAppend.js"));
+  config.addFilter("stringify", require("./filters/stringify.js"));
+  config.addFilter("capitalize", (text) =>
+    text.replace(/\b\w/g, (m) => m.toUpperCase())
+  );
 
   // Group posts and links into collections without leaning on tags
   config.addCollection("templates", async function (collection) {
     const templates = collection.getFilteredByGlob("src/site/template/*.md");
 
-    return await Promise.all(templates.map(async (template) => {
-      const { full_name } = parseGitUrl(template.data.repo);
-
-      template.data.stars = await fetch(`https://api.github.com/repos/${full_name}`, {
-        headers: {
-          Authorization: `token ${process.env.GITHUB_TOKEN}`
-        }
+    return await Promise.all(
+      templates.map(async (template) => {
+        const { full_name } = parseGitUrl(template.data.repo);
+
+        template.data.stars = await fetch(
+          `https://api.github.com/repos/${full_name}`,
+          {
+            headers: {
+              Authorization: `token ${process.env.GITHUB_TOKEN}`,
+            },
+          }
+        )
+          .then((res) => res.json())
+          .then((json) => json.stargazers_count);
+
+        return template;
       })
-        .then(res => res.json())
-        .then(json => json.stargazers_count);
-
-      return template;
-    }));
+    );
   });
 
+  // Add redirects file to root of output directory
+  config.addPassthroughCopy("_redirects");
```

Comment:
This is the main change to this file. The rest were done by prettier. I can revert if necessary!